src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingToAndFromManyAddressesSpecification_Steps.cs

```diff
@@ -93,18 +93,17 @@ private void Mine100Coins(CoreNode node)
 
         private void node2_receives_the_funds()
         {
+            // Need to mine a block prior to checking for spendable transactions in node2's wallet.
+            this.sharedSteps.MineBlocks(1, this.nodes[NodeOne], WalletAccountName, WalletName, WalletPassword);
+
             TestHelper.WaitLoop(() => this.nodes[NodeTwo].FullNode.WalletManager().GetSpendableTransactionsInWallet(WalletName).Any());
 
             this.nodeTwoBalance = this.nodes[NodeTwo].WalletBalance(WalletName);
 
             this.nodeTwoBalance.Should().Be(Money.Coins(50));
 
-            this.nodes[NodeTwo].WalletHeight(WalletName).Should().BeNull();
```

Comment:
This test is correct, because if the block hasn't been mined, the block height will be null.

Reply:
This is the wallet height, not the block height.

Reply:
the method is badly named but it checks the block height of the latest transaction.